lms/static/js/spec/main.js

```diff
@@ -703,6 +708,10 @@
         'js/spec/financial-assistance/financial_assistance_form_view_spec.js',
         'js/spec/groups/views/cohorts_spec.js',
         'js/spec/instructor_dashboard/certificates_bulk_exception_spec.js',
+        'js/spec/shoppingcart/shoppingcart_spec.js',
+        'js/spec/instructor_dashboard/instructor_dashboard_spec.js',
+        'js/spec/instructor_dashboard/ecommerce_spec.js',
+        'js/spec/instructor_dashboard/student_admin_spec.js',
```

Comment:
They do exist, but are further down in the file. So basically, you duplicated them.

In any case, your changes to this file will be reverted as part of deleting the unit test.